backend/mapservice/Controllers/ConfigController.cs

```diff
@@ -655,5 +665,59 @@ public string ListImage()
             }
             return JsonConvert.SerializeObject(fileList);
         }
+
+        public string ListVideo()
+        {
+            Response.Expires = 0;
+            Response.ExpiresAbsolute = DateTime.Now.AddDays(-1);
+            Response.ContentType = "application/json; charset=utf-8";
+            Response.Headers.Add("Cache-Control", "private, no-cache");
+
+            string tempPath = "/Upload";
+            string folder = Server.MapPath(tempPath);
+
+            IEnumerable<string> files = Directory.GetFiles(folder);
+            List<string> fileList = new List<string>();
+            foreach (string file in files)
+            {
+                string fileName = String.Empty;
+                if (Path.GetExtension(file).ToLower() == ".mp4" || Path.GetExtension(file).ToLower() == ".mov" || Path.GetExtension(file).ToLower() == ".ogg")
+                {
+                    fileName = Path.GetFileName(file);
+                    if (fileName != "layers")

```

Comment:
Why?

Reply:
To diffrate pictures from videos etc. that are in the same folder.

Reply:
I think @jacobwod is referring to the `if (fileName != "layers")` part. The configuration files are not in the same folder as the upload-data, right?

Reply:
Correct, the configuration files are not in the Upload folder, only data files such as images and videos. I guess this line is from an previous iteration of DOP.

Reply:
I will fix this issue and update the PR with that fix.

Reply:
Done.